todoapp/TODOapp/app/src/main/java/com/nearsoft/androidschool/todoapp/activities/map/MapsActivity.java

```diff
@@ -37,9 +38,10 @@ protected void onCreate(Bundle savedInstanceState) {
     public void onMapReady(GoogleMap googleMap) {
         map = googleMap;
 
-        LatLng latLng = new LatLng(-34, 151);
-        map.addMarker(new MarkerOptions().position(latLng).title(toDoContent.getTitle()));
-        map.moveCamera(CameraUpdateFactory.newLatLng(latLng));
+//        TODO get toDoLatLng from toDoContent variable
```

Comment:
The main idea of this task is for this to be interactive, you can just get the current location and [use it here without any trouble](https://developer.android.com/training/location/retrieve-current.html)